tests/FixerFactoryTest.php

```diff
@@ -398,6 +400,10 @@ public function testFixerDefinitions(FixerInterface $fixer)
                 }
             }
 
+            if ($sample instanceof VersionedCodeSample && !$sample->isSuitableForVersion(PHP_VERSION_ID)) {
```

Comment:
Do it before calling `$fixer->confgure(...)` (so just after `assertStringIsNotEmpty`), then tests will pass.